### ISSUE TITLE
Tried to flash from SD card?

### ISSUE THREAD

**Issue by probonopd** [RX309Electronics/Tapo-C200V3-cam-research-project#4]
Hello @RX309Electronics. If I read it correctly, you managed to damage your firmware and the watchdog reboots it all the time.

Have you tried to flash another firmware from the microSD slot?

Here is a description, but for a slightly different model:
https://github.com/nervous-inhuman/tplink-tapo-c200-re/issues/4#issuecomment-2267505153

https://github.com/tapo-firmware/Directory/blob/main/all_keys.txt
lists the names of known firmware files, maybe

http://download.tplinkcloud.com/firmware/Tapo_TC70v3_en_1.3.8_Build_230913_Rel.54885n_up_boot-signed_1695869685783.bin

would work (the "TC70v3" makes me believe so), but __no guarantees__.

**Comment by RX309Electronics** [comment]
I might have a look. Otherwise i might be able to make a firmware image by using the GPL code from TapoLink. I know the firmware is based on Openwrt And has a few custom binaries by tplink. TPLink has the buildtools and the project folders licensed under GPL on the opensource page and i believe they use buildroot

**Comment by RX309Electronics** [comment]
It does not work and my camera does not get to the state where it tests the motors... It just keeps saying service file /usr/sbin/uhttpd is not executable. i found a firmware repo for tapolink devices: https://github.com/tapo-firmware/Tapo_C200 and might look if i can use any of those files and if not i might try and see if they contain everything and if they do i will use my ch341 soic8 clip to flash it on the camera. Otherwise i might try to make a new rootfs with the buildtools.. Thanks for the suggestion though! I did manage to get it in a different mode because now it does not complain about the 'ispfile not ready' thingy. I found the pcb links that transmit the usb signals of the rtl8xxx wifi chip, and i might try to hook up a usb ethernet interface to it so it does not have to configure the wireless interface and i can boot over tftp maybe. The pcb pads that have to be bridged are near the chip near the 4r7 inductor

**Comment by probonopd** [comment]
@RX309Electronics were you able to revive your brick?

**Comment by RX309Electronics** [comment]
Yes i have been able to resolve the brick by installing the custom firmware (i was not even going to use the stock Firmware anyways. And with the added fact that the US is planning on abandoning TPLink due to multiple sloppy security measures and multiple vulnerabilities and backdoors), but i abandoned the project because of a lack of motivation and due to being busy with school/college and personal things. I do still have the camera and its up and running using thingino firmware streaming via rtsp to a local nas and i can access the footage through home assistant and vlc. 

Thank you for asking though :). I might pick up the project in the future when i graduate next year but for now i put it on a halt but i do have all the files for it!

**Comment by probonopd** [comment]
So how did you install Thingino on it?
Asking because the camera is not listed on https://thingino.com/.
